Classes/Interfaces/Hook.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+namespace OliverKlee\Seminars\Interfaces;
+
+/**
+ * Extend this interface to use Your interface with the HookService
```

Comment:
Extend -> implement

Reply:
This interface is NOT for implementation, it is the [base for other hook interfaces only](https://github.com/oliverklee/ext-seminars/blob/48e50e10472da840c6e6613973aa704e0b603d41/Classes/Interfaces/Hook/SeminarListView.php#L16).

Reply:
Ah, okay. Then it make sense. :-) Thanks for the explanation!